client/state/plugins/installed/selectors.js

```diff
@@ -120,6 +120,28 @@ export const getPlugins = createSelector(
 	}
 );
 
+export const getPluginsWithUpdateStatuses = createSelector(
```

Comment:
This one could use some tests.

Reply:
Sounds good! [Added it here](https://github.com/Automattic/wp-calypso/pull/95561/commits/8bdd9273fbab01ef80fbeb1eac90b05a30d3a1b9).